src/bramble/client/ProjectStats.js

```diff
@@ -0,0 +1,263 @@
+/* eslint
```

Comment:
Brackets already has an eslint config, see https://github.com/mozilla/brackets/blob/master/.eslintrc.json.  Why not just have this code use that?  My guess is that we aren't passing this file to eslint in the Gruntfile.